artview/components/correlation.py

```diff
@@ -642,12 +641,25 @@ def _update_plot(self):
                 self.Vradar.value, self.VfieldHorizontal.value,
                 self.VfieldVertical.value, sweeps, gatefilter, self.ax,
                 vmin + 0.05 * (vmax-vmin), vmax - 0.05 * (vmax-vmin),
-                str(self.horizontal_scale_menu_group.checkedAction().text()),
-                str(self.vertical_scale_menu_group.checkedAction().text()),
+                xscale,
+                yscale,
                 color=self.parameters["color"])
 
         self.canvas.draw()
 
+    def get_xyscale(self):
+        scale = [None, None]
+        scale[0] = str(self.horizontal_scale_menu_group.checkedAction().text())
+        scale[1] = str(self.vertical_scale_menu_group.checkedAction().text())
+        
+        for i in range(2):
+            if scale[i] == "&linear":
+                scale[i] = "linear"
+            elif scale[i] == "l&og":
+                scale[i] = "log"
+        return scale
```

Comment:
Will the `&` character always be in that position?
This [SO](https://stackoverflow.com/questions/3939361/remove-specific-characters-from-a-string-in-python) has a discussion of methods to remove an arbitrary string. Another discussion notes that even though the `.replace` method is techinically not the right way to do it, it is very fast.

```
scale[i].replace("&", "")
```